.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+
+on: [push]
+
+jobs:
+  lint:
+    name: Lint on ${{ matrix.node-version }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [10]
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, and pretest
+      run: |
+        npm install
+        npm run pretest
+      env:
+        CI: pretest
+
+  test:
+    name: Test on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [8, 10, 12]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        exclude:
+          - os: ubuntu-latest
+            node-version: 10
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, and test
+      run: |
+        npm install
+        npm run jest
+      env:
+        CI: true
+
+  coverage:
+    name: Test coverage on Node.js ${{ matrix.node-version }} and ${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        node-version: [10]
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, and test
+      run: |
+        npm install
+        npm run jest --coverage
+      env:
+        CI: true
```

Comment:
As I see `CI` variable does not mean anything, right now there is only one way -  pass `--ci` flag to jest. I could be wrong, will be great, if you point me out where it uses

There is PR about `Github Actions`, see https://github.com/watson/ci-info/pull/42 - `ci-info` used in `jest` to determine is current environment CI server/runner/worker

Reply:
Try to remove it.